internal/formats/cyclonedx13json/test-fixtures/snapshot/TestCycloneDxDirectoryPresenter.golden

```diff
@@ -1,10 +1,10 @@
 {
   "bomFormat": "CycloneDX",
   "specVersion": "1.3",
-  "serialNumber": "urn:uuid:a81dc685-cf22-48e0-bda5-65ea1a8bca5b",
+  "serialNumber": "urn:uuid:3305b0e9-bc4a-46f1-87fe-e9707cca5211",
```

Comment:
Is this going to be updated each time now? Should we include this and timestamp as redacted fields? I don't think this is something from your changes so maybe I can file a separate PR for this.

Reply:
I think the uuid is already redacted in testing https://github.com/anchore/syft/blob/main/internal/formats/cyclonedx13json/encoder_test.go#L32

Reply:
I had to update the golden snapshots due to the addition of new fields. As a consequence the uuids were updated when I ran the regeneration commands.

Reply:
@spiffcs idea for a follow-up PR: we could move the redactor calls to before the "update snapshot" call, therefor we wouldn't need to see UUIDs in the diff, we'd only see "redacted" which would never change.